libs/vscode-ui/components/src/lib/autocomplete/autocomplete.component.ts

```diff
@@ -114,6 +92,21 @@ export class AutocompleteComponent
       )
     ).pipe(startWith(false), debounceTime(300), shareReplay(1));
 
+    this.visibleOptions = this._options$.pipe(
+      switchMap((options) =>
+        this.control.valueChanges.pipe(
+          startWith(this.control.value || ''), // When panel is first opened, show the entire list
+          map(
+            (formValue) =>
+              options &&
+              options.filter((option) =>
+                option.toLowerCase().includes(formValue.toLowerCase())
+              )
+          ),
+        )
+      )
+    );
+
     this.selectedOption$ = this.isFocused$.pipe(
```

Comment:
`selectedOption$` is a little bit 😬 but I left it alone. I did remove the subscribe to it below though.

Reply:
Yea, that subscribe is a weird pattern..